### ISSUE TITLE
Typescript issue

### ISSUE THREAD

**Issue by lbelinsk** [kentor/flush-promises#4]
Hey,

I believe this pr broke our project: https://github.com/kentor/flush-promises/pull/2
I get the following typescript issue:

```
error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/Users/liorbe/WIX/wix-ui/packages/wix-ui-core/node_modules/flush-promises/index")' has no compatible call signatures.
```

In order to fix the issue on our side, instead of importing the default function from this module, I changed it:

```js
// not working
import * as flushPromises from 'flush-promises';

// working
const flushPromises = require('flush-promises');
```

I believe this is not intentional, so I can look into it in order to add a small fix.

Thanks...

**Comment by lbelinsk** [comment]
I started to look into it, and I already see that this pr is resolving the issue:
https://github.com/kentor/flush-promises/pull/3

I think this should be merged soon as this is breaking the users of this library.

**Comment by kentor** [comment]
yeah clearly i don't use typescript.

published @1.0.2
